### ISSUE TITLE
Make critical sections of code that could introduce vulnerability be reviewed by at least 2 person with .github/CODEOWNERS

### ISSUE THREAD

**Issue by hboon** [AlphaWallet/alpha-wallet-android#2878]
At least 2 person in case one of them is the author

i.e. modify the file https://github.com/AlphaWallet/alpha-wallet-android/blob/master/.github/CODEOWNERS

2 types of code should be covered:

* key management
* dapp browser, web3 provider injection and hook/callback handlers

Refactor, move a few files around if necessary?

iOS version https://github.com/AlphaWallet/alpha-wallet-ios/blob/master/.github/CODEOWNERS

**Comment by JamesSmartCell** [comment]
@hboon should we create a list of sensitive files, where touching them triggers this review?

**Comment by hboon** [comment]
@JamesSmartCell yes. A combination of directories and filenames works too. If you already stuff things in a tight Java package, maybe you’ll only need directories?
